diff --git a/third_party/blink/renderer/modules/media_capabilities/media_capabilities.h b/third_party/blink/renderer/modules/media_capabilities/media_capabilities.h
--- a/third_party/blink/renderer/modules/media_capabilities/media_capabilities.h
+++ b/third_party/blink/renderer/modules/media_capabilities/media_capabilities.h
@@ -157,6 +157,7 @@
     std::optional<bool> db_is_power_efficient;
     std::optional<bool> is_gpu_factories_supported;
     std::optional<bool> is_builtin_video_codec;
+    media::VideoCodec video_codec = media::VideoCodec::kUnknown;
     base::TimeTicks request_time;
     QueryType query_type;
 
